presto-main/src/test/java/com/facebook/presto/cost/TestComparisonStatsCalculator.java

```diff
@@ -155,6 +167,28 @@ public void setUp()
                 .build();
     }
 
+    private Consumer<SymbolStatsAssertion> equalTo(SymbolStatsEstimate estimate)
+    {
+        return symbolAssert -> {
+            symbolAssert
+                    .lowValue(estimate.getLowValue())
+                    .highValue(estimate.getHighValue())
+                    .distinctValuesCount(estimate.getDistinctValuesCount())
+                    .nullsFraction(estimate.getNullsFraction());
+        };
+    }
+
+    private SymbolStatsEstimate capNDV(SymbolStatsEstimate symbolStats, double rowCount)
+    {
+        // todo: add capping in test when logic actually does that
```

Comment:
is this coming soon?  I wonder if it's misleading to have tests that test "capNdv".  It might give the impression that it's supported, people might not check the method implementation to see that it's just a stub.

Reply:
Yes, this is the "normalizers" stuff. Already completed on the CBO branch with this method fully implemented.